tutorials-book/src/ch01-03-migration-api.md

```diff
@@ -0,0 +1,225 @@
+# Migration (API)
+
+If you prefer to set up and run the migrations programmatically, we provide the `Migrator` API for that.
+
+This section covers how to perform migrations without the need to install and use the CLI tool.
+
+## Preparation
+
+Add the cargo dependency `sea-orm-migration`:
+
+```diff
+
+...
+
+[dependencies]
+futures = "0.3.21"
+sea-orm = { version = "0.8.0", features = [ "sqlx-mysql", "runtime-async-std-native-tls", "macros" ] }
++ sea-orm-migration = "0.8.3"
+
+...
+
+```
```

Comment:
I'm a bit confused here. Should we specify the file path at the beginning, e.g. `Cargo.toml`?